.github/workflows/continuous-integration.yaml

```diff
@@ -109,6 +109,7 @@ jobs:
       needs-38-linux-runner: ${{ steps.set-matrix.outputs.needs-38-linux-runner }}
       needs-39-linux-runner: ${{ steps.set-matrix.outputs.needs-39-linux-runner }}
       needs-310-linux-runner: ${{ steps.set-matrix.outputs.needs-310-linux-runner }}
+      needs-311-linux-runner: ${{ steps.set-matrix.outputs.needs-311-linux-runner }}
```

Comment:
Have this been set properly in our GH secrets?

Reply:
I don't know what it is about, could someone do it please?

Reply:
My mistake

Reply:
shall we worry about the GH secrets @bcm-at-zama  ?

Reply:
I don't know, let's merge it